awx/main/isolated/manager.py

```diff
@@ -370,33 +370,32 @@ def health_check(self, instance_qs):
                 private_data_dir
             )
 
-            if runner_obj.status == 'successful':
```

Comment:
All this diff does is removes this `if runner_obj.status == 'successful':` check.